### PR TITLE
expression: refactor functions.

### PR DESCRIPTION
**Pull request by hanfei1991**
@shenli @coocood @tiancaiamao @zimulala PTAL

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

File: expression/builtin_string.go [hunk 1]
```diff
@@ -1150,16 +1070,12 @@ type builtinBitLengthSig struct {
 	baseBuiltinFunc
 }
 
-func (b *builtinBitLengthSig) eval(row []types.Datum) (types.Datum, error) {
+// https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_bit-length
```
Comment (zimulala) [inline expression/builtin_string.go]: Add `See`.

File: expression/builtin_string.go [hunk 2]
```diff
@@ -1184,16 +1100,12 @@ type builtinCharSig struct {
 	baseBuiltinFunc
 }
 
-func (b *builtinCharSig) eval(row []types.Datum) (types.Datum, error) {
+// https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_char
```
Comment (zimulala) [inline expression/builtin_string.go]: Ditto.

**Comment by zimulala** [comment]
Rest LGTM

**Comment by hanfei1991** [comment]
@zimulala PTAL

**Pull request merged**